### PR TITLE
Added html for section 1

### PR DESCRIPTION
**Pull request by Galin7777**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,30 @@
 </head>
 <body>
   <header>Шапка сайта</header>
-  <section>1</section>
+  <section class="section" id="warranty">
```
Comment (Palych18) [inline index.html:10]: You need to replace the id value with "download"

File: index.html [hunk 2]
```diff
@@ -7,7 +7,30 @@
...
+        <h1 class="title">
+          Peaceful shopping mindful money
+        </h1>
+        <p class="copy">
+          See how we can help with making your shopping experience and money
+          management more ZEN.
+        </p>
+        <div class="links">
+          <a href="https://apple.com" class="link">
+            <img src="./accets/icons/apple.svg" alt="apple icon">
```
Comment (Palych18) [inline index.html:22]: Recommended to close the single tag
```
<img src="./accets/icons/apple.svg" alt="apple icon" />
```

**Review by Palych18 — COMMENTED** [review]
Fix

File: index.html (hunk 1 above)
Comment (Galin7777) [reply index.html]: Поправила

File: index.html (hunk 2 above)
Comment (Galin7777) [reply index.html]: сделано)

**Review by Palych18 — APPROVED** [review]
Done

**Pull request merged**